hedyorg/hedy
# [BUG] Highlights commands in tip

Fixes #4234 also fixes #4247 by 1) not adding the prefix when we are translating code, and 2) updating the appended code so it does not use the `_` character. That means that it if leaks out again, at least kids will be able to run the code. 

**How to test**

Run the code given in #4234 and observe that the error message shows if highlighted instead of `{if}`.
Follow the steps in #4247 and observe that no appended code appears.
